includes/wikia/services/RevisionUpvotesService.class.php

```diff
@@ -204,9 +204,10 @@ public function getUserNewUpvotes( $userId ) {
 			return [];
 		}
 
-		$upvotes = $this->getUserUpvotes( $userId, $status['last_notified'] );
-
-		return $upvotes;
+		return $this->getUserUpvotes(
+			$userId,
+			!empty( $status['last_notified'] ) ? $status['last_notified'] : null
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "last_notified" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)